src/custom-elements/src/components/tabs/tabs.tsx

```diff
@@ -254,7 +252,7 @@ export class Tabs {
           const direction = this.vertical
             ? TabTriggerDirections.Top
             : TabTriggerDirections.Left;
-          const size = this.calculateSize(
+            const size = this.calculateSize(
```

Comment:
Please check the format of the whole file.

```suggestion
          const size = this.calculateSize(
```